department-of-veterans-affairs/abd-vro
# Task: remove cc tech debt

## What was the problem?

- remove unused code from developing contention classification API

Associated tickets or Slack threads:
- n/a

## How does this fix it?[^1]

Deleting stuff

## How to test this PR
- Make sure the CI still passes.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
